### PR TITLE
Added a tests for missing methods

### PR DESCRIPTION
**Pull request by justinpolygon**
Added a few missing methods. We should have really good coverage for models now.

### CODE REVIEW

File: rest/models/quotes_test.go [hunk 1]
```diff
@@ -29,6 +30,7 @@ func TestListQuotesParams(t *testing.T) {
 		WithTimestamp(models.LTE, timestamp).
 		WithTimestamp(models.GT, timestamp).
 		WithTimestamp(models.GTE, timestamp).
+		WithDay(year, month, day).
```
Comment (mmoghaddam385) [inline rest/models/quotes_test.go:33]: I don't think we should have this function and a `WithTimestamp(models.EQ, ...)` on the same model here, the `WithDay` shadows that call. In this case they're the same so the expectation doesn't change, but I'm not sure what we're really testing here

File: rest/models/trades_test.go [hunk 2]
```diff
@@ -29,6 +30,7 @@ func TestListTradesParams(t *testing.T) {
 		WithTimestamp(models.LTE, timestamp).
 		WithTimestamp(models.GT, timestamp).
 		WithTimestamp(models.GTE, timestamp).
+		WithDay(year, month, day).
```
Comment (mmoghaddam385) [inline rest/models/trades_test.go:33]: Same comment as above

File: rest/models/quotes_test.go (hunk 1 above)
Comment (justinpolygon) [reply rest/models/quotes_test.go]: Makes sense - removed the code.

File: rest/models/trades_test.go (hunk 2 above)
Comment (justinpolygon) [reply rest/models/trades_test.go]: Makes sense - removed the code.

**Comment by justinpolygon** [comment]
Closing for now - need to read the coverage action docs.

**Pull request closed without merging**